inspirehep/modules/converttohal/tei.py

```diff
@@ -0,0 +1,131 @@
+from jinja2 import Environment, PackageLoader
```

Comment:
Shall we call the whole module 'hal'? In this way we can put inside everything related to 'hal', not necessarily just conversion, but also pushing (and maybe pulling at some point).